src/Core/src/Hosting/AppHost.cs

```diff
@@ -0,0 +1,21 @@
+#nullable enable
+
+namespace Microsoft.Maui.Hosting
+{
+	public static class AppHost
+	{
+		public static IAppHost Current { get; internal set; } = null!;
```

Comment:
We can make this more secretive and add EditorVisible = Never on this...

Reply:
i would do that..